react/components/Table/Totalizers.js

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment, PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import TotalizerLabel from './totalizers/TotalizerLabel'
+import TotalizerValue from './totalizers/TotalizerValue'
+import TotalizerIcon from './totalizers/TotalizerIcon'
+
+const BORDER_COLOR = 'b--light-gray'
+
+const TOTALIZERS_CLASSES = `w-100 flex flex-column flex-row-ns ba br3 mb5 ${BORDER_COLOR}`
+const TOTALIZER_CLASSES = `flex flex-column flex-auto pa4 ${BORDER_COLOR}`
+
+class Totalizers extends PureComponent {
+  render() {
+    const { items } = this.props
+
+    if (items.length > 0) {
+      return (
+        <div id="totalizers" className={TOTALIZERS_CLASSES}>
```

Comment:
If used twice in the page the id will clash. Why is it needed?

Reply:
Good point and not useful.